content/docs/cucumber/state.md

```diff
@@ -91,14 +97,21 @@ World(MyHelper, MyOtherHelpers)
 This will `extend` each new `World` object with those modules.
 
 If you use [Ruby on Rails](/docs/tools/ruby#ruby-on-rails), there is already a `World` set up for you, so you will get
-an instance of `Cucumber::Rails::World`, which is a subclass of `ActionDispatch::IntegrationTest`. This gives you access to a lot of Rails' helper methods.
+an instance of `Cucumber::Rails::World`, which is a subclass of `ActionDispatch::IntegrationTest`. This gives you access
```

Comment:
I don't understand exactly what changed here. The verbiage looks the same. Maybe just a different line break? But I don't see a line break after "access" on the preview. I know this is just my own ignorance, sorry.

Reply:
I wrote that before I figured out what "polyglot" must mean. So the change is just to make these parts of the page dynamic dependent on the programming language selected, right?

Reply:
Yes. Polyglot means there are different programming languages on the page (as listed in the "front matter" at the top of the page) and we have dynamic content for each language using the "{{% block "ruby" %}}" and "{{% /block %}}" around a block of text specific for one language (ruby in this example), or a combination of languages (comma-separated).